test/hotspot/jtreg/gc/stress/TestStressRSetCoarsening.java

```diff
@@ -293,10 +293,7 @@ public void go() throws InterruptedException {
                 if (pre > cur) {
                     // Number of references went down.
                     // Need to provoke recalculation of RSet.
-                    WB.g1StartConcMarkCycle();
-                    while (WB.g1InConcurrentMark()) {
-                        Thread.sleep(1);
-                    }
+                    WB.g1RunConcurrentGC(false);
```

Comment:
probably use `WB.g1RunConcurrentGC();` overload for consistency with other calls